apps/snitch_core/priv/repo/migrations/20180312101809_add_stock_transfers_table.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Snitch.Repo.Migrations.AddStockTransfersTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:snitch_stock_transfers) do
+      add(:type, :string)
+      add(:reference, :string)
+      add(:source_location_id, references(:snitch_stock_locations))
+      add(:destination_location_id, references(:snitch_stock_locations))
```

Comment:
Since `:destination_location_id` and `:number` are "required", adding a `null: false` would make things rock solid.